Update upgrade-distribution-lists.md (#10562)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/manage/upgrade-distribution-lists.md

# Upgrade distribution lists to Microsoft 365 Groups in Exchange Online

> **_NOTE:_**  This functionality is going to be deprecated by 1st Feb, 2023.

Reviewer: ```suggestion
> [!NOTE]
> This functionality will be deprecated by February 1, 2023.
```

# Upgrade distribution lists to Microsoft 365 Groups in Exchange Online

> > [!Important]

Reviewer: ```suggestion
> [!IMPORTANT]
```

> > This functionality will be deprecated by February 1, 2023. For more information, see [Deprecating Conversion of Distribution Lists to Microsoft 365 Groups - Microsoft Community Hub](https://techcommunity.microsoft.com/t5/exchange-team-blog/deprecating-conversion-of-distribution-lists-to-microsoft-365/ba-p/3668215)

Reviewer: ```suggestion
> This functionality will be deprecated by February 1, 2023. For more information, see [Deprecating Conversion of Distribution Lists to Microsoft 365 Groups - Microsoft Community Hub](https://techcommunity.microsoft.com/t5/exchange-team-blog/deprecating-conversion-of-distribution-lists-to-microsoft-365/ba-p/3668215).
```
